.github/workflows/deployment.yml

```diff
@@ -117,11 +117,11 @@ jobs:
       - name: Build snap (1) Setup snapcraft
```

Comment:
I think it would make sense to run this in parallel as it's own job?

Reply:
Currently does not work at all: https://github.com/JabRef/jabref/actions/runs/49715346

    cannot read snap file "build/distribution/jabref*.snap"

Trying to debug.

Reply:
It's also very difficult to do it in parallel in a separate workflow (which is not what you meant, right?). IMHO it depends on the steps above (including jpackage, ...). Thus, we would need to duplicate 100 SLOC of the build file. 

We could work with different jobs in the same build file. Then, we would need to introduce uploading the `build` directory (more?) to GitHub as data passing between jobs can only be done that way. This is not comfortable (we have to be sure that we don't forget something)

Reply:
Cannot read is because the `*` is not expanded. Fixed it in cd707467ab5ffb7d5c00